packages/popup-component/src/components/popup/popup.scss

```diff
@@ -94,6 +94,29 @@
     color: #808080;
     font-size: $font-size-base * 0.375;
   }
+
+  &__action {
+    height: 3rem;
+    width: 11.25rem;
+    font-size: 1.25rem;
+    padding: .5rem 1rem;
+    border-radius: .3rem;
+    border: none;
+    margin-top: 2rem;
+    padding: .5rem 1rem;
+    font-weight: bold;
+    color: $white;
+    background:$light-green;
+    border: 1px solid $light-green;
```

Comment:
Any specific reason on why no use rems here? We encourage full usage of rems to keep proportions always accordingly to the user preferences

Reply:
There was no specific reason, I was not aware of using rems instead of px. I agree with you, it will keep proportions 🚀